checks/linux-local/docker-compose.yml

```diff
@@ -1,23 +1,23 @@
 version: '2'
 services:
-  teamcity-server:
-    image: teamcity-server:local-linux
-    ports:
-     - "8112:8111"
- 
+  #  teamcity-server:
+  #    image: teamcity-server:local-linux
+  #    ports:
+  #      - "8112:8111"
+
```

Comment:
Learning question: why `teamcity-server` and `teamcity-agent-sudo` on line#18 in this file is commented out in this PR?